Content.Shared/Inventory/Events/RefreshEquipmentHudEvent.cs

```diff
@@ -4,7 +4,7 @@ public sealed class RefreshEquipmentHudEvent<T> : EntityEventArgs, IInventoryRel
 {
     public SlotFlags TargetSlots { get; init; }
     public bool Active = false;
-    public object? ExtraData;
```

Comment:
This is better off being an object type in case anyone needs custom data.

Reply:
All custom data should be sourced from the components.